iquip/apps/scheduler.py

```diff
@@ -151,10 +151,7 @@ def updateInfo(self, info: ExperimentInfo):
             self.argsLayout.addWidget(QLabel(f"{key}: {value}", self))
 
     def priority(self) -> int:
-        """Returns the priority for sorting.
-
-        Returns:
-            The "priority" value of the experiment.
+        """Returns the "priority" value of the experiment.
         """
```

Comment:
It does not have to be here. When there is only the docstring summary, please write it in a single line: `"""Returns ... experiment."""`